Final-assignment_trial-and-error_8_linking classes_v2.py

```diff
@@ -0,0 +1,534 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Import modules
+
+# In[1]:
+
+
+import datetime #datetime package to convert strings into dates, calculate time periods etc.
+import re
+import json
+
+
+# # Freelancers
+
+# In[12]:
+
+
+class Freelancers:
+    is_internal = False
+    
+    def __init__(self, name, email, phone, ref):
+        self.name = name
+        self.email = email
+        self.phone = phone
+        self.ref = ref
+        
+        self.task = set()
+        self.language = set()
+    
+    def add_task(self, task):
+        self.task.add(task)
+    
+    def add_language(self, language):
+        self.language.add(language)
+    
+    def __str__(self):
+        sent_1 = f"{self.name} can be contacted via e-mail ({self.email}) or via phone ({self.phone})."
+        sent_2 = f"{self.name}'s ID in our database is {self.ref}."
+        sent_3 = f"{self.name} works as a {', '.join(self.task)} for {', '.join(self.language)}."
+        return "\n".join([sent_1, sent_2, sent_3])
+
+
+# In[2]:
+
+
+sdw = {
```

Comment:
This data should not be in the python script. It is only for illustration so it should be as a separate file or in the notebook.